examples/VRKit/post-processing/script.js

```diff
@@ -0,0 +1,87 @@
+import * as UTILS from '../../globals';
+
+const postprocessor = new WHS.PostProcessorModule();
+
+const world = new WHS.App([
+  new WHS.ElementModule(),
+  new WHS.SceneModule(),
+  new WHS.CameraModule(UTILS.appDefaults.camera),
+  new WHS.RenderingModule(UTILS.appDefaults.rendering, {
+    shadow: true
+  }),
+  new PHYSICS.WorldModule(UTILS.appDefaults.physics),
+  new WHS.OrbitControlsModule(),
+  new WHS.ResizeModule(),
+  new StatsModule(),
+  new VRKit.VRModule(),
+  // postprocessor
```

Comment:
@hirako2000 It will be used when we'll make post-processing work with `VREffect`.